package.json

```diff
@@ -18,23 +18,25 @@
   "tonicExampleFilename": "tonic-example.js",
   "scripts": {
     "clean": "rimraf dist",
-    "build": "npm run clean && npm run build:modules && npm run build:standalone",
+    "build": "npm run clean && npm run build:modules && npm run build:standalone && npm run build:types",
     "build:modules": "BABEL_ENV=modules babel lib -d dist/es-modules/",
     "build:standalone": "webpack && NODE_ENV=production webpack",
     "build:standalone:log": "NODE_ENV=production WEBPACK_MODE=log webpack --json --profile --progress > webpack-build-log.json && webpack-bundle-analyzer webpack-build-log.json",
-    "docs:build": "jsdoc -c jsdoc.json",
+    "build:types": "yarn tsc --declaration --noEmit false --emitDeclarationOnly true --allowJs false",
```

Comment:
Maybe clear the `generated` folder before emitting to ensure up-to-date state of type declarations.

Reply:
good idea!

Reply:
@marcolink added